### PR TITLE
Deploy website

### PR DESCRIPTION
**Pull request by Arch-Noize**
Deployed my portfolio website through the help of [GitHub Pages](https://docs.github.com/en/pages)!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 # 📖 Portfolio <a name="about-project"></a>
 
-- This website shows a sneak peek into the projects I have developed and deployed, showcasing my skills in HTML and CSS.With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
+- This website shows a sneak peek into the projects I have developed and deployed, showcasing my skills in HTML and CSS.With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites. You can find a live version of the website [here](https://arch-noize.github.io/portfolio-microverse/)!
```
Comment (Ibizugbe) [inline README.md]: Nice job deploying your project👍. But now you have deployed your project, I suggest you add the live demo section back to the readme and add the link to the deployed project to the live demo section😉😉.

File: styles.css [hunk 2]
```diff
@@ -68,7 +68,7 @@ button:active {
   width: 100%;
   align-items: center;
   padding: 45px;
-  background-image: url("/Images/header_illustration.png");
+  background-image: url("Images/header_illustration.png");
```
Comment (Ibizugbe) [inline styles.css]: Some of the images are missing in the live version of the project, kindly fix these issues as this is a requirement that the online version works properly😉.

image

image

image

Kindly take note of the absence of the background images👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Arch-Noize 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Project have been depolyed using github pages👍.
- [x] Github flow have been used👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Kindly ensure your commit messages are capitalized as this is a more professional practice😉.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdallahmalima — APPROVED** [review]
Hi @Arch-Noize , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Project deployed✔️
- Readme  file has live link✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**